index.html

```diff
@@ -359,12 +359,43 @@ <h3>
       additional members to control audio packetization.
     </p>
     <pre class="idl">partial dictionary RTCRtpEncodingParameters {
-  unsigned long ptime;
-  boolean adaptivePtime = false;
+    RTCResolutionRestriction scaleResolutionDownTo;
+    unsigned long            ptime;
+    boolean                  adaptivePtime = false;
 };</pre>
     <section id="rtcrtpencodingparameters-attributes">
       <h2>Dictionary {{RTCRtpEncodingParameters}} Members</h2>
       <dl data-link-for="RTCRtpEncodingParameters" data-dfn-for="RTCRtpEncodingParameters" class="dictionary-members">
+        <dt>
+          <dfn data-idl>scaleResolutionDownTo</dfn> of type <span class="idlMemberType">{{RTCResolutionRestriction}}</span>
+        </dt>
+        <dd>
+          <p>The maximum dimensions at which to restrict this encoding.</p>
+          <p>The default "scale resolution down by" factors (e.g. 4:2:1) are not
+          applied when this parameter is used. Instead, frames are either sent
+          as-is or they are downscaled by the encoder to uphold the
+          restrictions. The encoder will never upscale a frame. In simulcast,
+          not upscaling can result in multiple same-sized encodings if the size
+          of the frame is smaller than {{scaleResolutionDownTo}}; an encoding is
+          not sent if {{scaleResolutionDownTo}} is larger than the frame and
+          there exists another {{RTCRtpEncodingParameters/active}} encoding with
+          a smaller {{scaleResolutionDownTo}} density resulting in the same
+          size.</p>
+          <p>When setting parameters, if {{scaleResolutionDownTo}} is specified
```

Comment:
Maybe add a note this is both for addTransceiver and setParameters

Reply:
Done